Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/internal/Edge_collapse.h

```diff
@@ -313,7 +312,8 @@ class EdgeCollapse
     CGAL_expensive_assertion(data.is_in_PQ());
     CGAL_expensive_assertion(mPQ->contains(h));
 
-    data.set_PQ_handle(mPQ->update(h, data.queue_handle()));
+    mPQ->update(h);
+    data.set_is_in_PQ();
```

Comment:
is that necessary given it is an update and not a push?

Reply:
I think you're right